x-pack/metricbeat/module/mssql/os/data.go

```diff
@@ -0,0 +1,80 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package os
+
+import (
+	"time"
+
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	schema = s.Schema{
+		// Returns a miscellaneous set of useful information about the computer,
+		// and about the resources available to and consumed by SQL Server.
+		"info": s.Object{
+			"cpu_ticks":                      c.Int("cpu_ticks"),
```

Comment:
We may want to think on a more structured fields schema.

Reply:
Please define _more structured fields schema_ :smile: or describe the _unstructured_ nature of the current one to understand your reasoning

Reply:
I mean to group metrics in structured objects, like in the example with the memory fields I give in the other comment, or we may also group cpu metrics in:
```
cpu.ticks
cpu.count
...
```
I don't say we have to do some kind of regrouping for all fields, but think if it would make sense for some of them.